js/apps/system/_admin/aardvark/APP/frontend/js/views/loggerView.js

```diff
@@ -99,6 +101,25 @@
       this.logTopicView.render();
     },
 
+    downloadEntries: function () {
+      let toDownload = [];
+
+      // sort entries
+      this.fetchedEntries.sort(function compare(a, b) {
+        let dateA = new Date(a.date);
+        let dateB = new Date(b.date);
+        return dateA - dateB;
```

Comment:
I think we should also sort by a secondary field here, as there can be lots of log entries with the same date value.

Reply:
Done. Also, text/message now unescaped in download.